wandb/sdk/internal/internal_api.py

```diff
@@ -3205,6 +3204,7 @@ def use_artifact(
                     createdAt
                     labels
                     metadata
+                    _TTL_DURATION_SECONDS_
```

Comment:
I don't think the result of this function is ever used. This looks like a random subset of fields. Why did you add TTL here?

Reply:
Thanks thats a great catch, looks like this is only used for Launch https://github.com/wandb/wandb/pull/2622